Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/core/RecoveryCertificateRepository.kt

```diff
@@ -277,6 +277,13 @@ class RecoveryCertificateRepository @Inject constructor(
         }
     }
 
+    suspend fun replaceCertificate(
+        certificateToReplace: RecoveryCertificateContainerId,
```

Comment:
Wouldn't it be easier to identify both with the QR code? The reissuance data structure has the qr code

Reply:
Both are fine , at the end we will `find ` the certificate by either fields , `RecoveryCertificateContainerId ` has the `hash` anyway